pkg/odo/cli/project/project.go

```diff
@@ -35,7 +35,7 @@ func NewCmdProject(name, fullName string) *cobra.Command {
 		// 'odo project' is the same as 'odo project get'
 		// 'odo project <project_name>' is the same as 'odo project set <project_name>'
 		Run: func(cmd *cobra.Command, args []string) {
-			if len(args) > 0 && args[0] != "get" && args[0] != "set" {
+			if len(args) > 0 && args[0] != getRecommendedCommandName && args[0] != setRecommendedCommandName {
```

Comment:
Perhaps this should be changed to `RecommendedGetCommandName` like it's been done in `component.go`? I mean it doesn't need to be visible, but lets at least keep the same naming convention

Reply:
Actually, I will change the component names instead since all the other commands are using the `<command>RecommendedCommandName` format but I will do that in  #1282 instead.

Reply:
:+1: